modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/BlockTcpDiscoverySpi.java

```diff
@@ -74,8 +74,16 @@ private synchronized void apply(ClusterNode addr, TcpDiscoveryAbstractMessage ms
             clo.apply(addr, delegate);
     }
 
-    /** {@inheritDoc} */
-    @Override protected void writeToSocket(
+    /**
+     * Writes message to the socket.
+     *
+     * @param sock Socket.
```

Comment:
Why replaced the inherited doc?

Reply:
fixed. i've reviewd code obe more time and i've veririden method instead of overloading. so doc is inherited now.
TcpDiscoveryConnectionCheckMessage is used as a stub instead of msg which was pased via parameter according to your comment:
// Replace logic routine message with a stub to update last-sent-time to avoid segmentation on
// connRecoveryTimeout.